### ISSUE TITLE
ofxLua : glm bind issue

### ISSUE THREAD

**Issue by TheTechnobear** [danomatika/swig-openframeworks#23]
hi, 

ofxLua is not compiling (current openframeworks) on macOS, appears to be issues with glm 

seems to be with glm:fma

any suggestions on where to look?

thanks 
Mark

```
  "glm::vec<2, float, (glm::qualifier)0> glm::fma<glm::vec<2, float, (glm::qualifier)0> >(glm::vec<2, float, (glm::qualifier)0> const&, glm::vec<2, float, (glm::qualifier)0> const&, glm::vec<2, float, (glm::qualifier)0> const&)", referenced from:
      _wrap_fma(lua_State*) in glmBindings.o
  "glm::vec<3, float, (glm::qualifier)0> glm::fma<glm::vec<3, float, (glm::qualifier)0> >(glm::vec<3, float, (glm::qualifier)0> const&, glm::vec<3, float, (glm::qualifier)0> const&, glm::vec<3, float, (glm::qualifier)0> const&)", referenced from:
      _wrap_fma(lua_State*) in glmBindings.o
  "glm::vec<4, float, (glm::qualifier)0> glm::fma<glm::vec<4, float, (glm::qualifier)0> >(glm::vec<4, float, (glm::qualifier)0> const&, glm::vec<4, float, (glm::qualifier)0> const&, glm::vec<4, float, (glm::qualifier)0> const&)", referenced from:
      _wrap_fma(lua_State*) in glmBindings.o
ld: symbol(s) not found for architecture x86_64
```

**Comment by danomatika** [comment]
What is "current openframeworks"? Current _release version_ or git master? I generally don't test against master as it's a moving target.

**Comment by TheTechnobear** [comment]
current released version , so 0.11.2

(I did test against night build, **as well**, and that is the same)

**Comment by danomatika** [comment]
It's most likely some glm header include has changed or that particular set of functions were renamed or removed. Simplest approach would be to edit the glm.i SWIG interface file in the `swig` folder & try regenerating the bindings.

**Comment by TheTechnobear** [comment]
yeah, i tried to regenerate bindings .. but honestly, i wasnt sure exactly that the interface file was looking for as a function signature in the glm header.  (theres alot of macros involved ;)) 

i also tried to just comment out the fmt entry in the interface file, and then re-generate binding, but oddly seemed to get the same issue.
that said, its my first time messing with swig, so likely i screwed something up.

**Comment by Jonathhhan** [comment]
Hi @TheTechnobear did you find a solution for that issue meanwhile?

**Comment by TheTechnobear** [comment]
unfortunately, no... I did try, but I was still getting same issue.... and then really had to get on with some other projects , so had to leave it.

**Comment by Jonathhhan** [comment]
I understand. I tried a lot without luck, but I am more an artist than a programmer, so it is also a bit too complex for me. I think for now I will compile Ofelia (that is the main reason why I use ofxLua) with OF 0.10.1 and ofxLua 1.3.7, I do not think that I have any drawbacks with that. Maybe thats also of interest regarding the issue: https://openframeworks.cc/learning/02_graphics/how_to_use_glm/

**Comment by danomatika** [comment]
Fixed by 8eed144775cdaf44f00418b4893b4386c1b7044d
